layer_v3/pyproject.toml

```
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "aws-lambda-powertools-python-layer"
+version = "1.1.0"
+description = "Powertools for AWS Lambda (Python) Lambda Layers"
+authors = ["DevAx <aws-devax-open-source@amazon.com>"]
```

```suggestion
authors = ["Powertools for AWS Maintainers <aws-powertools-maintainers@amazon.com>"]
```

Might be worth checking for other entries with DevAx instead of Powertools maintainers team

---

Good catch! Fixed